app/api/index.js

```diff
@@ -9,3 +13,423 @@ export const TEAMS: string = '/teams';
+        id: number,
+        name: string,
+        surname: string,
+        number: string,
+        image: string,
+      },
+      player_2_home: ?{
+        id: number,
+        name: string,
+        surname: string,
+        number: string,
+        image: string,
+      },
+      player_1_away: {
+        id: number,
+        name: string,
+        surname: string,
+        number: string,
+        image: string,
+      },
+      player_2_away: ?{
+        id: number,
+        name: string,
+        surname: string,
+        number: string,
+        image: string,
+      },
+      goals_home: number,
+      goals_away: number,
+    },
+  },
+};
+
+export type GetTeamResponse = {
+  id: number,
+  name: string,
+  image: ?string,
+  table: ?string,
+  league: ?{
+    id: number,
+    name: string,
+    shifting_rule: boolean,
+    cup: boolean,
+  },
+  venue: ?{
+    id: number,
+    name: string,
+    street: string,
+    zip_code: number,
+    city: string,
+  },
+  position: ?number,
+  home_match_day: ?number,
+  home_match_time: ?string,
+  club: ?{
+    id: number,
+    name: string,
+  },
+  player: Array<{
+    id: number,
+    name: string,
+    surname: string,
+    number: string,
+    image: string,
+  }>,
+};
+
+export type GetTeamMatchesResponse = Array<{
+  id: number,
+  datetime: number,
+  team_home: {
+    id: number,
+    name: string,
+    image: ?string,
+  },
+  team_away: {
+    id: number,
+    name: string,
+    image: ?string,
+  },
+  venue: {
+    id: number,
+    name: string,
+    street: string,
+    zip_code: number,
+    city: string,
+  },
+  league: {
+    id: number,
+    name: string,
+    shifting_rule: boolean,
+    cup: boolean,
+  },
+  set_points: boolean,
+  set_points_home: ?number,
+  set_points_away: ?number,
+  goals_home: ?number,
+  goals_away: ?number,
+  live: boolean,
+  match_day: string,
+  score_suggest: ?number,
+  score_unconfirmed: number,
+  date_confirmed: boolean,
+}>;
+
+export type GetPlayerResponse = {
+  id: number,
+  name: string,
+  surname: string,
+  number: string,
+  image: string,
+  classification: ?string,
+  international_number: string,
+  association: {
+    id: number,
+    name: string,
+    short: string,
+  },
+  club: {
+    id: number,
+    name: string,
+  },
+  ranglists: Array<{
+    id: number,
+    name: string,
+    season: string,
+    position: number,
+    participants: number,
+  }>,
+  tournament_participations: Array<{
+    id: number,
+    name: string,
+    discipline: string,
+    date: number,
+    location: string,
+    position: number,
+    participants: number,
+  }>,
+  teams: Array<{
+    id: number,
+    name: string,
+    season: string,
+    competitions: Array<string>,
+  }>,
+  statistics: Array<{
+    name: string,
+    wins: number,
+    draws: number,
+    lost: number,
+    matches: number,
+    position: number,
+    competitive_index: number,
+    rate: string,
+  }>,
+};
+
+export type AuthenticateResponse = {};
```

Comment:
@arnef can you help me with these response types? I haven't had the chance to see them in the console yet, so I can only guess what's in there ;)